.github/workflows/octokit.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           - 2.7
           - "3.0"
           - "3.1"
+          - 3.2
```

Comment:
We should probably standardize on quotes here in this matrix - I'm good with either way - @petergoldstein let me know if that is something you'd like to do here, if not no worries - I can do a follow-up to get it handled.

Reply:
The "3.0" requires quotes - otherwise it gets truncated to "3", resulting in the latest Ruby 3 getting loaded.  The others don't.  But I'm happy to quote everything.